### DISCUSSION TITLE
Thoughts on how to improve the UI?

### DISCUSSION THREAD

**Discussion by balzack** [balzack/databag#61] [Show and tell]
The next big feature I plan to work on is an update of the mobile UI. Here are my initial thoughts on how it can be improved, but if you have any other thoughts please let me know.
 
@samuraikid0 (tagging you because you've been quite involved in getting it to this point)

[light_mode](https://github.com/balzack/databag/assets/96387156/527e36b3-4e17-447c-91bb-1e352c72e948)
[dark_mode](https://github.com/balzack/databag/assets/96387156/4733ea1a-ca78-4728-ba43-12763712c0eb)

**Comment by samuraikid0** [comment]
interesting didn't receive this notification, just saw it by casuality
Thanks ! It's an awesome piece of open source

**Reply by balzack** [reply]
Thank you! You were critical in getting it this far.
